tools/nni_gpu_tool/gpu_metrics_collector.py

```diff
@@ -25,9 +25,13 @@
 from xml.dom import minidom
 
 def check_ready_to_run():
-    #TODO check process in windows
     if sys.platform == 'win32':
-        return True
+        pgrep_output = subprocess.check_output('wmic process where "CommandLine like \'%nni_gpu_tool.gpu_metrics_collector%\' and name like \'%python%\'" get processId')
```

Comment:
Could you describe the process and motivation of 'check ready to run'?

Reply:
It will check whether the nni_gpu_tool.gpu_metrics_collector is running or not excluding the self process. if it return false, it will not continue to get information and write to gpu_metrics.

Reply:
Got it.